framework/src/mysql.js

```diff
@@ -239,6 +239,15 @@ const getTableInstance = async (tableName, tableConfig, connEndpoint = CONN_ENDP
 			query.whereIn(property, values);
 		}
 
+		if (params.whereIns) {
+			const { whereIns } = params;
+
+			whereIns.forEach(param => {
+				const { property, values } = param;
+				query.whereIn(property, values);
+			});
+		}
```

Comment:
Please remove this section for the `whereIns` and update the block for `whereIn` to be generic like below and adjust the tests accordingly:

```
		if (params.whereIn) {
			const whereIns = Array.isArray(params.whereIn)
				? params.whereIn
				: [params.whereIn];

			whereIns.forEach(param => {
				const { property, values } = param;
				query.whereIn(property, values);
			});
		}
```

Reply:
Applied.